From 307d4f83c052290cfdfef09608b785e4dfd0da78 Mon Sep 17 00:00:00 2001
From: Marouen Ghodhbane <marouen.ghodhbane@nxp.com>
Date: Tue, 25 Apr 2023 10:54:02 +0200
Subject: [PATCH 50/58] configs: arm64: add the required architecture
 information in cell creation helpers

- Per f2320d2ee4, the architecture field is now required. Otherwise the following error would happen:

[   37.582074] The Jailhouse is opening.
[   37.591773] jailhouse: Configuration architecture mismatch

Signed-off-by: Marouen Ghodhbane <marouen.ghodhbane@nxp.com>
---
 configs/arm64/cell-create.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/arm64/cell-create.h b/configs/arm64/cell-create.h
index 087e1928..8be18809 100644
--- a/configs/arm64/cell-create.h
+++ b/configs/arm64/cell-create.h
@@ -44,6 +44,7 @@ struct {
 	.cell = {
 		.signature = JAILHOUSE_CELL_DESC_SIGNATURE,
 		.revision = JAILHOUSE_CONFIG_REVISION,
+		.architecture = JAILHOUSE_ARM64,
 		.name = CONFIG_INMATE_NAME,
 		.flags = JAILHOUSE_CELL_PASSIVE_COMMREG | JAILHOUSE_CELL_VIRTUAL_CONSOLE_ACTIVE,
 
-- 
2.34.1

